src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -500,13 +500,15 @@ export class ResultsViewPageStore {
                 return anySelected;
             },
 
+            set customBinary(val: boolean) {
```

Comment:
maybe time improve this name.   have no idea what customBinary is.  i realize this is legacy code

Reply:
@alisman  I agree that the name is very uninformative. `customBinary` refers to _custom_ diver annotations that can be used or not-used for VUS filtering, hence `Binary`. Refactoring of names is realy needed here, yes.